### ISSUE TITLE
Problem with anaconda install on Linux

### ISSUE THREAD

**Issue by kadrlica** [mgckind/easyaccess#77]
I tried upgrading easyaccess to 1.3.0 using anaconda on Linux and I get this error when trying to start the executable or import then module:

```
Traceback (most recent call last):
  File "/cvmfs/des.opensciencegrid.org/fnal/anaconda2/envs/default/bin/easyaccess", line 37, in <module>
    from version import __version__
ImportError: No module named version
```

This is most likely a misleading error message and the real problem is in the preceding `import` block within the `try` statement (see [here](https://github.com/mgckind/easyaccess/blob/fa33a60a488e43c43f27e2007280a86ace16e3f4/easyaccess/easyaccess.py#L26)).

I've downloaded the source and I found out that there was an error being thrown by `fitsio` when easyacccess tried to import it:

```
/lib64/libc.so.6: version `GLIBC_2.14' not found (required by /cvmfs/des.opensciencegrid.org/fnal/anaconda2/envs/default/lib/python2.7/site-packages/fitsio/_fitsio_wrap.so)
```

**Comment by kadrlica** [comment]
I reverted to `fitsio-0.9.8rc2` and everything works ok. So this appears to be an issue with the anaconda build of `fitsio 0.9.8`.

**Comment by mgckind** [comment]
I saw this issue in somebody else computer (Andres Plazas), I tried in 2 linux distros and worked fine.  I'm planning a 1.3.1 version soon (after i finish some documentation) , and we can revert fitsio back to rc2 for that version, or figure out the problem with Erin Sheldon.

**Comment by mgckind** [comment]
is it a 32-bit linux distro?

**Comment by mgckind** [comment]
I think is a 32/64 bit issue by looking at the commit history. There is a fix later in master, but not in the released version, we can built  a anaconda version directly from master for 1.3.1

**Comment by kadrlica** [comment]
It is a 64bit distro of Scientific Linux (on the Fermilab cluster).

```
LSB Version:    :base-4.0-amd64:base-4.0-noarch:core-4.0-amd64:core-4.0-noarch:graphics-4.0-amd64:graphics-4.0-noarch:printing-4.0-amd64:printing-4.0-noarch
Distributor ID: ScientificFermi
Description:    Scientific Linux Fermi release 6.4 (Ramsey)
Release:    6.4
Codename:   Ramsey
```

I could also try to build `fitsio 0.9.8` on this local machine and see how it goes. If it works then I can package it to anaconda.

**Comment by kadrlica** [comment]
I think this is related (and possibly solved) in #99.

**Comment by mgckind** [comment]
I'm closing this one as #99 will solve the same issue
